### PR TITLE
Feature/open alex

### PR DESCRIPTION
**Pull request by TylerLackey**
https://rally1.rallydev.com/#/?detail=/userstory/796657868771&fdp=true

### CODE REVIEW

File: data/vocab/lite.ttl [hunk 1]
```diff
@@ -321,6 +321,33 @@ bld:rightsStatement a rdf:Property ;
     rdfs:range rdfs:Literal ;
     rdfs:domain bld:Resource .
 
+bld:type a rdf:Property ;
```
Comment (TylerLackey) [inline data/vocab/lite.ttl:324]: Remove

File: data/vocab/lite.ttl [hunk 2]
```diff
@@ -321,6 +321,33 @@ bld:rightsStatement a rdf:Property ;
...
+    rdfs:subPropertyOf bld:date ;
+    rdfs:domain bld:Resource ;
+    rdfs:range rdfs:Literal .
+
+bld:updated a rdf:Property ;
+    rdfs:label "updated"@en ;
+    rdfs:comment "A boolean value that represents whether or not a Resource has been updated."@en ;
+    rdfs:domain bld:Resource ;
+    rdfs:range rdfs:Literal .
+
```
Comment (TylerLackey) [inline data/vocab/lite.ttl:350]: Remove

File: data/vocab/lite.ttl [hunk 3]
```diff
@@ -321,6 +321,33 @@ bld:rightsStatement a rdf:Property ;
...
+    rdfs:subPropertyOf bld:date ;
+    rdfs:domain bld:Resource ;
+    rdfs:range rdfs:Literal .
+
+bld:updatedDate a rdf:Property ;
+    rdfs:label "updated date"@en ;
+    rdfs:comment "The date a Resource was last updated."@en ;
+    rdfs:subPropertyOf bld:date ;
+    rdfs:domain bld:Resource ;
+    rdfs:range rdfs:Literal .
```
Comment (TylerLackey) [inline data/vocab/lite.ttl:343]: Check to see if subProperty inherents the range from date. If it does remove the range reference here

File: data/vocab/lite.ttl [hunk 4]
```diff
@@ -576,6 +603,12 @@ bld:subOrganization a rdf:Property ;
     rdfs:domain bld:Organization ;
     rdfs:range bld:Resource .
 
+bld:researchOrganizationRegistry a rdf:Property ;
+	rdfs:label "research organization registry"@en ;
+	rdfs:comment "The link to the organization's research organization registry page. "@en ;
+    rdfs:domain bld:Organization ;
+    rdfs:subPropertyOf bld:link .
```
Comment (TylerLackey) [inline data/vocab/lite.ttl:610]: Should follow this example

scholar:orcid a rdf:Property ;
    rdfs:label "ORCID"@en ;
    rdfs:comment "An ORCID persistent digital identifier for a researcher."@en ;
    rdfs:domain bf:Person ;
	rdfs:range link:ORCID  .

Comment (TylerLackey) [reply data/vocab/lite.ttl:610]: Need to add the ROR class as well. This will be referenced in the ROR property

File: data/vocab/scholar.ttl [hunk 5]
```diff
@@ -437,6 +437,24 @@ scholar:discipline a rdf:Property ;
...
+    rdfs:label "works count"@en ;
+    rdfs:comment "The number of works associated with a person."@en ;
+    rdfs:range rdfs:Literal ;
+    rdfs:domain bf:Person .
+
+scholar:countsByYear a rdf:Property ;
+    rdfs:label "counts by year"@en ;
+    rdfs:comment "The number of works associated with a person organized by year."@en ;
+    rdfs:subClassOf bf:List .
+    rdfs:domain bf:Person .
```
Comment (TylerLackey) [inline data/vocab/scholar.ttl:450]: Cannot be a subclass of a class. Need to be changed to literal. Should be able pass in several instances doesn't have to be defined as a list

File: data/vocab/scholar.ttl [hunk 6]
```diff
@@ -437,6 +437,24 @@ scholar:discipline a rdf:Property ;
...
+
+scholar:worksLink a rdf:Property ;
+    rdfs:label "works link"@en ;
+    rdfs:comment "The link to the works associated with a person."@en ;
+    rdfs:range bf:link ;
+    rdfs:domain bf:Person .
```
Comment (TylerLackey) [inline data/vocab/scholar.ttl:456]: Need to remove. Property is being dropped within Glue Job

File: data/vocab/lite.ttl [hunk 7]
```diff
@@ -321,6 +321,33 @@ bld:rightsStatement a rdf:Property ;
     rdfs:range rdfs:Literal ;
     rdfs:domain bld:Resource .
 
+bld:type a rdf:Property ;
+    rdfs:label "type"@en ;
+    rdfs:comment "Text string representing the type of resource."@en ;
+    owl:equivalentProperty dc:type ;
+    rdfs:range rdfs:Literal ;
+    rdfs:domain bld:Resource .
```
Comment (gloriagonz) [inline data/vocab/lite.ttl:329]:
```suggestion
```

File: data/vocab/lite.ttl [hunk 8]
```diff
@@ -321,6 +321,27 @@ bld:rightsStatement a rdf:Property ;
...
+    rdfs:comment "The date a Resource was last updated."@en ;
+    rdfs:subPropertyOf bld:date ;
+    rdfs:domain bld:Resource ;
+    rdfs:range rdfs:Literal .
+
+bld:updated a rdf:Property ;
+    rdfs:label "updated"@en ;
+    rdfs:comment "A boolean value that represents whether or not a Resource has been updated."@en ;
+    rdfs:domain bld:Resource ;
+    rdfs:range rdfs:Literal .
```
Comment (gloriagonz) [inline data/vocab/lite.ttl:343]:
```suggestion
```

File: data/vocab/lite.ttl [hunk 9]
```diff
@@ -321,6 +321,21 @@ bld:rightsStatement a rdf:Property ;
     rdfs:range rdfs:Literal ;
     rdfs:domain bld:Resource .
 
+
+bld:createdDate a rdf:Property ;
```
Comment (gloriagonz) [inline data/vocab/lite.ttl:325]:
```suggestion
bld:createdDate a rdf:Property ;
```

File: data/vocab/lite.ttl [hunk 10]
```diff
@@ -576,6 +590,12 @@ bld:subOrganization a rdf:Property ;
     rdfs:domain bld:Organization ;
     rdfs:range bld:Resource .
 
+bld:researchOrganizationRegistry a rdf:Property ;
+	rdfs:label "research organization registry"@en ;
+	rdfs:comment "The link to the organization's research organization registry page. "@en ;
+    rdfs:domain bld:Organization ;
+    rdfs:subPropertyOf bld:link .
```
Comment (gloriagonz) [inline data/vocab/lite.ttl:597]:
```suggestion
    bld:ror a rdf:Property ;
    rdfs:label "research organization registry ID"@en ;
    rdfs:comment "An organization's Research Organization Registry open persistent identifier."@en ;
    rdfs:domain bld:Organization ;
    rdfs:range rdfs:Literal  .
```

File: data/vocab/lite.ttl [hunk 11]
```diff
@@ -576,6 +590,12 @@ bld:subOrganization a rdf:Property ;
     rdfs:domain bld:Organization ;
     rdfs:range bld:Resource .
 
+    bld:ror a rdf:Property ;
+    rdfs:label "research organization registry ID"@en ;
+    rdfs:comment "An organization's Research Organization Registry open persistent identifier."@en ;
+    rdfs:domain bld:Organization ;
+    rdfs:range rdfs:Literal  .
+
```
Comment (gloriagonz) [inline data/vocab/lite.ttl:598]:
```suggestion
```

Comment (gloriagonz) [reply data/vocab/lite.ttl:598]: ror id property should be in scholar not lite

File: data/vocab/scholar.ttl [hunk 12]
```diff
@@ -436,6 +436,29 @@ scholar:discipline a rdf:Property ;
     rdfs:comment "A field of study or academic discipline related to a person."@en ;
     rdfs:range bf:Concept ;
     rdfs:domain bf:Person .
+    bld:ror a rdf:Property ;
```
Comment (gloriagonz) [inline data/vocab/scholar.ttl:439]:
```suggestion

     bld:ror a rdf:Property ;
```

File: data/vocab/scholar.ttl [hunk 13]
```diff
@@ -437,6 +437,30 @@ scholar:discipline a rdf:Property ;
     rdfs:range bf:Concept ;
     rdfs:domain bf:Person .
 
+     bld:ror a rdf:Property ;
```
Comment (gloriagonz) [inline data/vocab/scholar.ttl:440]:
```suggestion
    bld:ror a rdf:Property ;
```

File: data/vocab/scholar.ttl [hunk 14]
```diff
@@ -437,6 +437,30 @@ scholar:discipline a rdf:Property ;
...
+scholar:worksCount a rdf:Property ;
+    rdfs:label "works count"@en ;
+    rdfs:comment "The number of works associated with a person."@en ;
+    rdfs:range rdfs:Literal ;
+    rdfs:domain bf:Person .
+
+scholar:countsByYear a rdf:Property ;
+    rdfs:label "counts by year"@en ;
+    rdfs:comment "The number of works associated with a person organized by year."@en ;
+    rdfs:subClassOf bf:List .
```
Comment (gloriagonz) [inline data/vocab/scholar.ttl:455]:
```suggestion
    rdfs:range rdfs:Literal ;
```

**Comment by gmreburn** [comment]
This branch has syntax issues from `bld:` vs `lite:` namespace usage. I will push fixes. We also have a consistency issue between namespace usage of `bld:` and `lite:` in these vocab files. We need to align - could be a separate PR.

File: data/vocab/scholar.ttl [hunk 15]
```diff
@@ -437,6 +437,30 @@ scholar:discipline a rdf:Property ;
     rdfs:range bf:Concept ;
     rdfs:domain bf:Person .
 
+bf:ror a rdf:Property ;
```
Comment (gloriagonz) [inline data/vocab/scholar.ttl:440]:
```suggestion
scholar:ror a rdf:Property ;
```

File: data/vocab/scholar.ttl [hunk 16]
```diff
@@ -437,6 +437,30 @@ scholar:discipline a rdf:Property ;
     rdfs:range bf:Concept ;
     rdfs:domain bf:Person .
 
+scholar:ror a rdf:Property ;
+    rdfs:label "research organization registry ID"@en ;
+    rdfs:comment "An organization's Research Organization Registry open persistent identifier."@en ;
+    rdfs:domain bf:Organization ;
```
Comment (gloriagonz) [inline data/vocab/scholar.ttl:443]:
```suggestion
    rdfs:domain scholar:Organization ;
```

File: data/vocab/scholar.ttl [hunk 17]
```diff
@@ -437,6 +437,30 @@ scholar:discipline a rdf:Property ;
...
+    rdfs:label "counts by year"@en ;
+    rdfs:comment "The number of works associated with a person organized by year."@en ;
+    rdfs:range rdfs:Literal ;
+    rdfs:domain bf:Person .
+
+scholar:worksLink a rdf:Property ;
+    rdfs:label "works link"@en ;
+    rdfs:comment "The link to the works associated with a person."@en ;
+    rdfs:range bf:link ;
+    rdfs:domain bf:Person .
```
Comment (gloriagonz) [inline data/vocab/scholar.ttl:462]:
```suggestion
```

**Review by gloriagonz — APPROVED** [review]
We discussed these changes, made updates based on our conversation and they're good to go!

**Pull request merged**